gsplat/project_gaussians.py

```diff
@@ -47,12 +47,13 @@ def project_gaussians(
        clip_thresh (float): minimum z depth threshold.
 
     Returns:
-        A tuple of {Tensor, Tensor, Tensor, Tensor, Tensor, Tensor}:
+        A tuple of {Tensor, Tensor, Tensor, Tensor, Tensor, Tensor, Tensor}:
 
         - **xys** (Tensor): x,y locations of 2D gaussian projections.
         - **depths** (Tensor): z depth of gaussians.
         - **radii** (Tensor): radii of 2D gaussian projections.
         - **conics** (Tensor): conic parameters for 2D gaussian.
+        - **compensation** (Tensor): the density compensation for blurring 2D kernel
```

Comment:
This extra return would break backward compatibility. Personally I'm fine with it as we are in active-developing version `0.1.x`. But I'll let @vye16  to decide starting from when we want to maintain backward compatibility.

Reply:
Hi @vye16 , could you help take a look at this PR and see if you have any other comments other than @liruilong940607